.github/workflows/ci_cd.yml

```diff
@@ -458,7 +458,7 @@ jobs:
           command: |
             export LD_LIBRARY_PATH=${{ env.ANSYSEM_ROOT241 }}/common/mono/Linux64/lib64:${{ env.ANSYSEM_ROOT241 }}/Delcross:$LD_LIBRARY_PATH
             source .venv/bin/activate
-            pytest -n 4 --dist loadfile --durations=50 -v --cov=pyaedt --cov-report=xml --cov-report=html --junitxml=junit/test-results.xml _unittest
+            pytest -n 2 --dist loadfile --durations=50 -v --cov=pyaedt --cov-report=xml --cov-report=html --junitxml=junit/test-results.xml _unittest
```

Comment:
@maxcapodi78 Do we have issue on some VMs when running the tests with 4 CPUs?

Reply:
@MaxJPRey We faced some random fails in Linux in the last PRs, we think this could be the issue.

Reply:
OK, good to know. Thanks @Samuelopez-ansys !